src/web/modules/custom/workbc_jobboard/src/Controller/apiServices.php

```
@@ -0,0 +1,153 @@
+      }
+      // set URL and other appropriate options
+		  curl_setopt($ch, CURLOPT_URL, $url);
+		  curl_setopt($ch, CURLOPT_CONNECTTIMEOUT ,0);
+		  curl_setopt($ch, CURLOPT_TIMEOUT, 800); //timeout in seconds
+
+		  if (!empty($header)) {
+        curl_setopt($ch, CURLOPT_HTTPHEADER, $header);
+		  }
+		  else {
+        curl_setopt($ch, CURLOPT_HEADER, "Content-Type:application/xml");
+		  }
+
+		  if($trace == true){
+        curl_setopt($ch, CURLOPT_STDERR, fopen('php://output', 'w'));
+        curl_setopt($ch, CURLOPT_VERBOSE, 1);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 0);
+		  }
+		  else{
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+      }
+
+		  curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, 0);
+
+		  if($trace == true){
+        curl_setopt($ch, CURLOPT_HEADER, 1);
+      }
+		  else{
+        curl_setopt($ch, CURLOPT_HEADER, 0);
+      }
+
+		  curl_setopt($ch, CURLOPT_FOLLOWLOCATION, TRUE);
+
+		  if(!empty($cookie)){
+        curl_setopt($ch, CURLOPT_COOKIE, $cookie);
+        curl_setopt($ch, CURLOPT_AUTOREFERER, true);
+        curl_setopt($ch, CURLOPT_COOKIESESSION, 1);
+		  }
+
+		  //try & catch for curl request to get url
+		  try {
+        // grab URL and pass it to the browser
+        $ret = curl_exec($ch);
+        $httpcode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+        
+        if($httpcode == 200 ){
+          if($get_vars == true){
+            $info = curl_getinfo($ch);
+            if(isset($info['url'])){
+              $url = parse_url($info['url']);
+
+              parse_str($url['query'], $r);
+
+              $r['query_string'] = $r;
+            }
+          }
+
+          if($ret_cookies == true){
+            // get cookie
+            preg_match('/^Set-Cookie:\s*([^;]*)/mi', $ret, $m);
+            $r['cookies'] = $m;
+          }
+          $r['response'] = $ret;
+          
+          return $r;
+        }
+        return false;
+		  }
+		  catch (Exception $e) {
+        return false;
```

Please log exceptions to `\Drupal::logger` instead of silently ignoring them.